network/linphone-desktop/linphone-desktop.SlackBuild

```diff
@@ -0,0 +1,401 @@
+#                  mscodec2-30d88f890897a8d82a4b195591045e9fe5cf7208 \
+#                  msoboe-11d0f703482ddc11d9d15ec81ed8c97d2d9bd317 \
+#                  msopenh264-5c7a9522478f5eedc9a3e1600fdd705da1e00d23 \
+#                  mssilk-ff9ddf985266bf25ba2dcd3256172fa0ab52e4b1 \
+#                  mswasapi-c8a851cc1358b860a93b42c17b267d1d8f6952b8 \
+#                  mswebrtc-80b8e33b11516870b887b80279b9da4de8b26ebe \
+#                  mswinrtvid-384dba6ff7ac37f4123f1f02b9b7b2bc40784dc4 \
+#                  ortp-6e05272d70456ef8a84684572a3bb847873ec176 \
+#                  postquantumcryptoengine-0f71ffb829102cce337ed5cdc42c5a57d06791ab\
+#                  webrtc-583acd27665cfadef8ab03eb85a768d308bd29dd)
+
+packagenames=( qtkeychain \
+                 ispell \
+                 linphone-sdk \
+                 bcg729 \
+                 bcmatroska2 \
+                 bctoolbox \
+                 bcunit \
+                 belcard \
+                 belle-sip \
+                 belr \
+                 bzrtp \
+                 aom \
+                 bv16-floatingpoint \
+                 codec2 \
+                 dav1d \
+                 decaf \
+                 ffmpeg \
+                 gsm \
+                 jsoncpp \
+                 libjpeg-turbo \
+                 liboqs \
+                 libvpx \
+                 libxml2 \
+                 libyuv \
+                 mbedtls \
+                 opencore-amr \
+                 openh264 \
+                 openldap \
+                 opus \
+                 soci \
+                 speex \
+                 sqlite3 \
+                 srtp \
+                 vo-amrwbenc \
+                 xerces-c \
+                 zlib \
+                 zxing-cpp \
+                 liblinphone \
+                 lime \
+                 mediastreamer2 \
+                 msaaudio \
+                 msamr \
+                 msandroidcamera2 \
+                 mscodec2 \
+                 msoboe \
+                 msopenh264 \
+                 mssilk \
+                 mswasapi \
+                 mswebrtc \
+                 ortp \
+                 postquantumcryptoengine\
+                 oboe \
+                 webrtc)
+
+
+for i in "${!modulepaths[@]}" ; do
+  echo "${!modulepaths[@]}"
+  tar xvf "$CWD"/"${packagenames[i]}"*
+  rmdir "${modulepaths[i]}"
+  ln -s "$(pwd)"/"${packagenames[i]}"* "${modulepaths[i]}"
+done
+git init >> /dev/null
+git config user.email "dummy@example.test"
+git config user.name "Example"
+git add README.md
+git commit -m "$PKGVERSION" > /dev/null
+git tag -m "$PKGVERSION" "$PKGVERSION"
+echo ""
+
+
+chown -R root:root .
+find -L . \
+ \( -perm 777 -o -perm 775 -o -perm 750 -o -perm 711 -o -perm 555 \
+  -o -perm 511 \) -exec chmod 755 {} + -o \
+ \( -perm 666 -o -perm 664 -o -perm 640 -o -perm 600 -o -perm 444 \
+  -o -perm 440 -o -perm 400 \) -exec chmod 644 {} + || true
+
+my_parallel="${MAKEFLAGS##-j}"
```

Comment:
Urchlay's pattern is probably better to use :

https://github.com/SlackBuildsOrg/slackbuilds/blob/6d80312a18250e30f10a3f68e533c0e55a2ec72a/games/pybik/pybik.SlackBuild#L89-L98